requirements.txt

```diff
@@ -12,6 +12,8 @@ setuptools~=58.0.4
 optax
 rich
 dataclasses~=0.6
+pyzmq  # requires libzmq3-dev to work
```

Comment:
Should we ship SwarmRL with resobee as a default, or do we add it as an optional package?

Reply:
I think optional would be better.